erpnext/accounts/doctype/share_transfer/share_transfer.py

```diff
@@ -12,7 +12,7 @@
 class ShareDontExists(ValidationError): pass
 
 class ShareTransfer(Document):
-	def before_save(self):
+	def before_submit(self):
 		if self.transfer_type == 'Issue':
 			company_doc = self.get_shareholder_doc(self.company)
```

Comment:
Can you rename the variable "company_doc" to "shareholder", because it is actually not company_doc